src/Core/Silk.NET.Core/Loader/LibraryLoader.cs

```diff
@@ -3,7 +3,7 @@
 // You may modify and distribute Silk.NET under the terms
 // of the MIT license. See the LICENSE file for details.
 
-#if NETCOREAPP3_0
```

Comment:
Why target 5.0 when we can target 3.0 and make more users happy?

Reply:
won't net5.0 users be unhappy then?

Reply:
`net5.0` == `netcoreapp5.0` remember? `netcoreapp3.0`'s TFM means any version above `3.0`, including .NET 5.

Reply:
Ok, thanks. Rider doesn't understand this yet.